merkato/merkato.py

```diff
@@ -54,16 +76,31 @@ def __init__(self, configuration, coin, base, spread, bid_reserved_balance, ask_
             last_order = get_last_order(self.mutex_UUID)
             new_history = get_new_history(current_history, last_order)
             self.rebalance_orders(new_history)
+
         self.initialized = True  # to avoid being updated before orders placed
 
+
+    def kill():
+        ''' TODO: Function comment
+        '''
+        # Cancel all orders
+        self.cancelrange(ONE_SATOSHI, ONE_BITCOIN) # Technically not all, but should be good enough
+
+        # Remove reserve references in DB
```

Comment:
shouldn't this remove the relevant merkato from the db and then cancel all orders?

Reply:
Due to how we're handling partials, we need to sum up all the partial fills of all active orders before cancelling them, and then remove that amount from the partials balance.

Reply:
Why? If all the orders are cancelled it shouldn't matter. No additional trades will be placed or execute and the merkato will be wiped.

Reply:
right of course, you're correct